include/WCSimDetectorConstruction.hh

```diff
@@ -80,7 +82,8 @@ public:
   G4int    GetMyConfiguration()   {return myConfiguration;}
   G4double GetGeo_Dm(G4int);
   G4int    GetTotalNumPmts() {return totalNumPMTs;}
-  
+  G4int    GetTotalNumPmtsOD() {return totalNumPMTsOD;}
```

Comment:
I’d suggest renaming this to `GetTotalNumODPmts` and `totalNumODPMTs`, to be consistent with `Get_ODPmts` (and `ComputeWCODPMT`, `GetODTubeID` _etc._)